client/app/components/Pages/OnlineJudge/AddQuestion.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class AddQuestion extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+
+        };
+    }
+    render() {
+        const styles = {
+            page: {
+                width: "90%",
+                background: "#E4ECF4",
+                marginLeft: "5%",
+                borderRadius: "10px",
+                marginTop:"20px",
+                marginBottom:"40px"
+            },
+            form: {
+                padding: "30px",
+            },
+            submitBtn:{
+                width:"20%",
+                marginLeft:"40%"
+            },
+            cancelBtn: {
+                width: "20%",
+                marginLeft: "40%",
+                marginTop:"20px"
+            }
+        }
+
+        return (
+            <div style={styles.page}>
+                <div style={styles.form}>
+                    <div class="input-group mb-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text">Problem Title</span>
+                        </div>
+                        <input type="text" class="form-control" aria-label="Problem Title" aria-describedby="basic-addon2" />
+                    </div>
+                    <div class="input-group mb-3">
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text">Problem Description</span>
+                            </div>
+                            <textarea class="form-control" aria-label="Problem Description"></textarea>
+                        </div>
+                    </div>
```

Comment:
Can each of these be a Component? Like a TextBox component. It will make it easier to edit later.